### PR TITLE
PurchasableCertificate + MarketUser

### PR DESCRIPTION
**Pull request by josipbagaric**
- Separates all "purchasable" properties, such as price and currency from the bare `Certificate` and creates a separate Entity called `PurchasableCertificate` with purchasable properties. 
- Use `MarketUser` instead of regular `User` for marketplace specific settings

### CODE REVIEW

File: packages/market/src/blockchain-facade/PurchasableCertificate.ts [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+    currency: Currency;
+}
+
+export const getCertificateListLength = async (
+    configuration: Configuration.Entity
+): Promise<number> => {
+    return Certificate.getCertificateListLength(configuration);
+};
+
+// const getAccountFromConfiguration = (configuration: Configuration.Entity) => ({
```
Comment (e00dan) [inline packages/market/src/blockchain-facade/PurchasableCertificate.ts]: Commented out code shouldn't be part of the commit

**Review by kosecki123 — APPROVED** [review]

**Review by e00dan — APPROVED** [review]
